### PR TITLE
easy-rsa: switch to release tarball

### LINKED ISSUE #180761: easy-rsa does not show real version, date, and git commit hash
### `brew config` AND `brew doctor` output

```shell
$ brew config
HOMEBREW_VERSION: 4.3.14
ORIGIN: https://github.com/Homebrew/brew
HEAD: 39d51a45e89ddb4572494c845522509e3deeeb61
Last commit: 3 days ago
Core tap JSON: 10 Aug 10:09 UTC
Core cask tap JSON: 10 Aug 10:09 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vi
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ANALYTICS: set
HOMEBREW_NO_ENV_HINTS: set
Homebrew Ruby: 3.3.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: octa-core 64-bit skylake
Clang: 14.0.0 build 1400
Git: 2.46.0 => /usr/local/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 12.7.6-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2

$ brew doctor
Your system is ready to brew.
```

### Verification

- [X] My `brew doctor` output says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

I tried to find out which version of _easyRSA_ has been installed by _homebrew_.

### What happened (include all command output)?

```
$ easyrsa --version
EasyRSA Version Information
Version:     ~VER~
Generated:   ~DATE~
SSL Lib:     OpenSSL 3.3.1 4 Jun 2024 (Library: OpenSSL 3.3.1 4 Jun 2024)
Git Commit:  ~GITHEAD~
Source Repo: https://github.com/OpenVPN/easy-rsa

$ easyrsa show-crl
Using Easy-RSA 'vars' configuration:
* /Users/example/.local/secrets/easyrsa/pki/vars

EasyRSA version ~VER~

Error
-----
No such 'crl' type file with a <file_name_base> of 'crl'.
Expected to find this file at:
* /Users/example/.local/secrets/easyrsa/pki/crl.pem
```

### What did you expect to happen?

- `Version` should be a real version—not `~VER~`, i.e. it contains the version of the easyRSA release (tag)
- `Generated` field should have a real date—not `~DATE~`
- `Git Commit` field should have the real git commit hash—not `~GITHEAD~`, i.e. it contains the git commit hash of the easyRSA release (tag)

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
$ brew install easy-rsa
$ easyrsa --version
$ easyrsa show-crl
```

### PR DESCRIPTION
**Pull request by gromgit**
`easyrsa` script in release tarball doesn't require version substitution, and has all the other required assets except the `KNOWN_ISSUES` doc file. I think that can be skipped since upstream didn't think it important enough to include anyway.

Closes #180761.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/e/easy-rsa.rb
Comment (chenrui333) [inline Formula/e/easy-rsa.rb:1]: maybe also update test to check the output as well?

File: Formula/e/easy-rsa.rb [hunk 1]
```diff
@@ -19,7 +19,12 @@ class EasyRsa < Formula
   depends_on "openssl@3"
 
   def install
-    inreplace "easyrsa3/easyrsa", "'/etc/easy-rsa'", "'#{pkgetc}'"
+    inreplace "easyrsa3/easyrsa" do |s|
+      s.gsub! "'/etc/easy-rsa'", "'#{pkgetc}'"
+      s.gsub! "~VER~", version.to_s
+      s.gsub! "~DATE~", time.iso8601.to_s
+      s.gsub! "~GITHEAD~", "RELEASE"
```
Comment (carlocab) [inline Formula/e/easy-rsa.rb:26]: Might be easier to use the release tarball at https://github.com/OpenVPN/easy-rsa/releases/download/v3.2.0/EasyRSA-3.2.0.tgz that includes these values, so we don't need to `inreplace` them.

Comment (gromgit) [reply Formula/e/easy-rsa.rb:26]: Huh, that's probably a better solution. I assuming switching sources like this warrants a revision bump?

Comment (carlocab) [reply Formula/e/easy-rsa.rb:26]: Probably not, TBH. We can revision bump it if someone else complains about it.

File: Formula/e/easy-rsa.rb
Comment (gromgit) [reply Formula/e/easy-rsa.rb:1]: I'm going with @carlocab's suggestion to use the release tarball, where the `easyrsa` script already has the necessary values in place, so I think we can skip this.

**Review by carlocab — APPROVED** [review]

**Pull request merged**